octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po

```diff
@@ -5720,6 +5724,26 @@ msgstr "Daemon offline"
 msgid "Reset daemon"
 msgstr "Dämon zurücksetzen"
 
+#~ msgid "QuickstartGuide_en.pdf"
```

Comment:
Is there a reason there are removed here? Are we doing the translation for multiple tickets?